paper-ieee-trans.tex

```diff
@@ -315,15 +315,16 @@ \subsection{EGI--Integrated Sustainable Pan--European Infrastructure}
 was the continuation of the {\sl EGEE--III} project, with the objective of establishing and
 maintaining a sustainable European Grid Infrastructure, composed by a federation of National
 Grid Initiatives and interoperable with other Grids worldwide.
-This goal would be accomplished through the
-development and maintenance of operational tools, such as the Operations Portal \cite{egi-ops},
-the {\sl EGI} Helpdesk \cite{ggus}, or the Grid Operations Configuration DataBase (GOCDB) \cite{gocdb}, and the
-management of the software provisioning process \cite{mario}, to validate and distribute the software to
-the production infrastructure. As a result, this process led to a production--ready
+This has been accomplished through the
```

Comment:
I prefer the conditional here, like it is in master version

Reply:
I sugested because it seem to me a conditional for a passed project, and with the next phrase wasn't very clear if from conditional in the end it was applied. But I can be wrong. Fill free to change/maintain as in master.

Reply:
not clear this comment, but I trust you

Reply:
leaving conditional